extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -313,33 +320,39 @@ public function sendConfirmationEmail() {
 			$this->errParam = '';
 
 			if ( $this->wg->Request->wasPosted() ) {
-				$action = $this->request->getVal( 'action','' );
-				if ( $action=='resendconfirmation' ) {
+				$action = $this->request->getVal( 'action', '' );
+				if ( $action == 'resendconfirmation' ) {
 					$response = $this->userLoginHelper->sendConfirmationEmail( $this->username );
 					$this->result = $response['result'];
 					$this->msg = $response['msg'];
 					$this->heading = wfMessage( 'usersignup-confirmation-heading-email-resent' )->escaped();
-				} else if ( $action == 'changeemail' ) {
-					$this->email = $this->request->getVal( 'email', '' );
-					$params = [
-						'username' => $this->username,
-						'email' => $this->email
-					];
-
-					$response = $this->sendSelfRequest( 'changeUnconfirmedUserEmail', $params );
-
-					$this->result = $response->getVal( 'result','' );
-
-					if( $this->result == 'ok' ) {
-						$this->msg = $response->getVal( 'msg','' );
-						$this->heading = wfMessage( 'usersignup-confirmation-heading-email-resent' )->escaped();
-					} else if( $this->result == 'error' ) {
-						$this->msgEmail = $response->getVal( 'msg','' );
-						$this->errParam = $response->getVal( 'errParam', '' );
-					} else if ( $this->result == 'confirmed' ) {
-						$this->heading = wfMessage( 'usersignup-confirm-page-heading-confirmed-user' )->escaped();
-						$this->subheading = wfMessage( 'usersignup-confirm-page-subheading-confirmed-user' )->escaped();
-						$this->msg = $response->getVal( 'msg','' );
+				} else {
```

Comment:
Yeah, intellij did this automatically after I added google styling for java... sigh!